src/main/kotlin/util/MinecraftTemplates.kt

```diff
@@ -66,6 +66,13 @@ class MinecraftTemplates : FileTemplateGroupDescriptorFactory {
             spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE_SETTINGS_GRADLE_TEMPLATE))
             spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE_POM_TEMPLATE))
             spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE_SUBMODULE_POM_TEMPLATE))
+            spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE8_MAIN_CLASS_TEMPLATE))
+            spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE8_BUILD_GRADLE_TEMPLATE))
+            // spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE8_SUBMODULE_BUILD_GRADLE_TEMPLATE))
+            spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE8_GRADLE_PROPERTIES_TEMPLATE))
+            spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE8_SETTINGS_GRADLE_TEMPLATE))
+            // spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE8_POM_TEMPLATE))
+            // spongeGroup.addTemplate(FileTemplateDescriptor(SPONGE8_SUBMODULE_POM_TEMPLATE))
```

Comment:
Why are these commented out?

Reply:
I simply forgot to remove them. Thought I needed different POM templates when I started working on this, but the current ones work fine.